src/styles/index.scss

```
@@ -1,3 +1,132 @@
+$size-box: 200px;
+$pad-box: 16px;
+$col-b: #00acdc;
+$size-img: calc($size-box - $size-box / 5);
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  padding-left: $pad-box;
+  padding-right: $pad-box;
+  display: flex;
+  flex-direction: column;
+  width: $size-box;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: $size-img;
+    height: 134px;
+    margin: 32px auto 40px;
+    display: block;
+  }
+
+  &__text1 {
+    color: #060b35;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__text2 {
+    color: #616070;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    margin-top: 4px;
+  }
+
+  &__review {
+    margin-top: 18px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    &-stars {
+      display: flex;
+      flex-direction: row;
+      gap: 7.2px;
+      margin-left: 1.3px;
+    }
+
+    &-text {
+      color: #060b35;
+      font-size: 10px;
+      line-height: 14px;
+      font-weight: 400;
+    }
+    &-star {
+      width: 12.8px;
+      height: 12.8px;
+      background-size: cover;
+      background-image: url("../images/star.svg");
+
+      &--active {
+        background-image: url('../images/star-active.svg'); // Активная звезда
```

Move stars styles to the separate file
```suggestion
        background-image: url('../images/star-active.svg');
```